tests/providers/cncf/kubernetes/operators/test_kubernetes_pod.py

```diff
@@ -914,10 +914,8 @@ def test_mark_checked_if_not_deleted(self, mock_patch_already_checked, mock_dele
         mock_delete_pod.assert_not_called()
 
 
-def test__suppress():
-    with patch("logging.Logger.error") as mock_error:
+def test__suppress(caplog):
+    with _suppress(ValueError):
+        raise ValueError("failure")
 
-        with _suppress(ValueError):
-            raise ValueError("failure")
-
-        mock_error.assert_called_once_with("failure", exc_info=True)
+    assert "ValueError: failure" in caplog.text
```

Comment:
using caplog is a better test because it doesn't depend precisely on the code structure, doesn't require mocking -- just looks at the log output